src/js_ast.zig

```diff
@@ -8677,6 +8677,14 @@ pub const ServerComponentBoundary = struct {
                 bun.unsafeAssert(l.list.capacity > 0); // optimize MultiArrayList.Slice.items
                 return l.list.items(.reference_source_index)[i];
             }
+
+            pub fn bitSet(scbs: Slice, alloc: std.mem.Allocator, input_file_count: usize) !bun.bit_set.DynamicBitSetUnmanaged {
+                var scb_bitset = try bun.bit_set.DynamicBitSetUnmanaged.initEmpty(alloc, input_file_count);
```

Comment:
It would be better to use AutoBitSet here if possible. That will hold the memory inline if it fits.

Reply:
i dont think autobitset is useful when you can just use stack-fallback allocator, which will hold the memory on the stack and avoid extra branches when reading into the bitset